transmogrifier/sources/json/aardvark.py

```diff
@@ -86,19 +88,106 @@ def get_optional_fields(self, source_record: dict) -> dict | None:
         # locations
 
         # notes
+        fields["notes"] = self.get_notes(source_record) or None
 
         # publication_information
+        fields["publication_information"] = (
+            self.get_publication_information(source_record) or None
+        )
 
-        # related_items
+        # related_items not used in MITAardvark
 
         # rights
+        fields["rights"] = self.get_rights(source_record) or None
 
         # subjects
         fields["subjects"] = self.get_subjects(source_record) or None
 
         # summary field
+        fields["summary"] = source_record.get("dct_description_sm")
+
         return fields
 
+    @staticmethod
+    def get_alternate_titles(source_record: dict) -> list[timdex.AlternateTitle]:
+        """Get values from source record for TIMDEX alternate_titles field."""
+        alternate_titles = []
+
+        if "dct_alternative_sm" in source_record:
+            for title_value in [
+                title_value for title_value in source_record["dct_alternative_sm"]
+            ]:
+                alternate_titles.append(timdex.AlternateTitle(value=title_value))
+
+        return alternate_titles
+
+    @staticmethod
+    def get_contributors(source_record: dict) -> list[timdex.Contributor]:
+        """Get values from source record for TIMDEX contributors field."""
+        contributors = []
+
+        if "dct_creator_sm" in source_record:
+            for contributor_value in [
+                contributor_value
+                for contributor_value in source_record["dct_creator_sm"]
+            ]:
+                contributors.append(
+                    timdex.Contributor(value=contributor_value, kind="Creator")
+                )
```

Comment:
Won't repeat for all, but wonder if this has some of the same potential for collapsing re: list comprehensions.